cnd/src/http_api/routes/rfc003/handlers/action.rs

```diff
@@ -8,11 +8,11 @@ use crate::{
         route_factory::new_action_link,
         routes::rfc003::decline::{to_swap_decline_reason, DeclineBody},
     },
+    init_swap::init_accepted_swap,
```

Comment:
Hey Thomas I've see you bring functions into scope a couple of times just recently, is this a result of that lint we discussed?  I was under the impression that idiomatic Rust only brings types and traits into scope but keeps one layer of module on function calls.  At least that is what the Rust book says (source link omitted due to laziness).